src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -1138,6 +1161,9 @@ public unsafe void ClearLogEntriesAndSetLastTruncate(TransactionOperationContext
         }
         public unsafe void TruncateLogBefore(TransactionOperationContext context, long upto)
         {
+            return;
```

Comment:
This looks like debug code that wasn't removed

Reply:
oops :/